### PR TITLE
test: add `TEST_REPSITORY_NAME_PREFIX` to isolate test recordings

### PR DESCRIPTION
**Pull request by gr2m**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # used for recording fixtures
+TEST_REPSITORY_NAME_PREFIX=
```
Comment (tmelliottjr) [inline .env.example:2]:
```suggestion
TEST_REPOSITORY_NAME_PREFIX=
```

File: test/recorded/_lib/create-test-repository.js [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 /**
+ * @param {NodeJS.ProcessEnv} env
  * @param {import("@octokit/core").Octokit} octokit
  * @param {string} owner
+ * @param {string} testName
  * @returns {Promise<import("@octokit/openapi-types").components["schemas"]["repository"]>}
  */
-export async function createRepository(octokit, owner) {
+export async function createRepository(env, octokit, owner, testName) {
+  const prefix = env.TEST_REPSITORY_NAME_PREFIX;
```
Comment (tmelliottjr) [inline test/recorded/_lib/create-test-repository.js:11]:
```suggestion
  const prefix = env.TEST_REPOSITORY_NAME_PREFIX;
```

File: test/recorded/_lib/delete-all-test-repositories.js [hunk 3]
```diff
@@ -1,9 +1,13 @@
 // @ts-check
 
 /**
+ * @param {NodeJS.ProcessEnv} env
  * @param {InstanceType<typeof import("./octokit").default>} octokit
  */
-export async function deleteAllTestRepositories(octokit, owner) {
+export async function deleteAllTestRepositories(env, octokit, owner) {
+  const prefix = env.TEST_REPSITORY_NAME_PREFIX
+    ? `${env.TEST_REPSITORY_NAME_PREFIX}-test-`
```
Comment (tmelliottjr) [inline test/recorded/_lib/delete-all-test-repositories.js:9]:
```suggestion
  const prefix = env.TEST_REPOSITORY_NAME_PREFIX
    ? `${env.TEST_REPOSITORY_NAME_PREFIX}-test-`
```

**Review by tmelliottjr — CHANGES_REQUESTED** [review]
Found a small typo.

**Comment by gr2m** [comment]
> Found a small typo.

Great catch! Fixed. Looking good otherwise?

**Pull request merged**